README.md

```diff
@@ -58,6 +71,11 @@ To get this app to work for you, you will need to have a Trello account and stor
   * API_TOKEN  -  Generate your own API token and store value here
   * BOARD_ID  -  The ID of the board you want to use for the project
 ```
+For this application:
+```bash
+my_statuses = ["Not Started", "In Progress", "Completed"]
+```
```

Comment:
Good job for documenting this btw!

Reply:
I have pushed v2 versions of the container and component diagram to the repo, please review and feedback.